package.json

```diff
@@ -195,8 +195,6 @@
     "changelog": "changelog nock all -m > CHANGELOG.md"
   },
   "pre-commit": [
-    "jshint",
```

Comment:
please don't remove these. If the tests are not passing, or the coverage decreased, please fix it. :)